phpseclib/Math/BigInteger/Engines/BCMath/Base.php

```diff
@@ -60,7 +60,7 @@ public static function isValidEngine()
      * @param string $class
      * @return \phpseclib\Math\BigInteger\Engines\BCMath
      */
-    protected static function powModHelper(BCMath $x, BCMath $e, BCMath $n, $class)
+    public static function powModHelper(BCMath $x, BCMath $e, BCMath $n, $class)
```

Comment:
You made a similar change in https://github.com/phpseclib/phpseclib/commit/f314f9795f2a5c43fc680cd6fee2beaf3e192239#r131527744 . I don't understand the motivation for that change nor do I understand it for this change.